### PR TITLE
Add about docker to README.md

### PR DESCRIPTION
**Pull request by youpong**
## What is this Pull Request About?

Add following procedures about docker to README.md

- Building the docker image
- Running searchor on the docker container

## What will this Pull Request Affect?

Update README.md

# Any additional information?

There was a plan to update README.md in #134, but it seems to be missing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +53,23 @@ Take a look at more examples in the [examples](https://github.com/ArjunSharda/Se
 
 *Note*:&nbsp; Engine names follow the **UpperCamelCase** convention.(eg: ChromeWebStore).
 
+Docker
+------
+
+Building the docker image
+```
+$ docker build -t searcher .
```
Comment (ArjunSharda) [inline README.md:61]:
```suggestion
$ docker build -t searchor .
```

File: README.md [hunk 2]
```diff
@@ -53,6 +53,23 @@ Take a look at more examples in the [examples](https://github.com/ArjunSharda/Se
...
+```
+
+Running searcher on the docker container
+```
+$ docker run --rm -it searcher bash
```
Comment (ArjunSharda) [inline README.md:66]:
```suggestion
$ docker run --rm -it searchor bash
```

File: README.md [hunk 3]
```diff
@@ -53,6 +53,23 @@ Take a look at more examples in the [examples](https://github.com/ArjunSharda/Se
...
 
+Docker
+------
+
+Building the docker image
+```
+$ docker build -t searcher .
+```
+
+Running searcher on the docker container
```
Comment (ArjunSharda) [inline README.md:64]:
```suggestion
Running searchor on the docker container
```

**Review by ArjunSharda — CHANGES_REQUESTED** [review]
Hi! Thanks for the PR. There seems to be some typos, so I have requested the changes for them.

**Comment by youpong** [comment]
Sorry for that. I fixed them.

**Review by ArjunSharda — CHANGES_REQUESTED** [review]
I think you have to remove the root part. Sorry for late reply, thought it sent.

**Comment by youpong** [comment]
I think that root part will be useful for who want to develop and test Searchor in a docker container.

**Comment by ArjunSharda** [comment]
> I think that root part will be useful for who want to develop and test Searchor in a docker container.

Ah, then okay you can keep it.

**Review by ArjunSharda — APPROVED** [review]
LGTM, thanks for the PR!

**Pull request merged**